OasisDEX/oasis-borrow
# 🔵 [frontend] Aave V3 Base - open/manage + portfolio entries

Changes:
- [Aave V3] added Base products
- [Aave V3] added required contracts/tokens
- [Aave V3] added portfolio integration

How to test:
- set up a base fork
- get funds
- open one of the aave v3 base positions
- check portfolio - the position should be there
- view the position
